### ISSUE TITLE
Quarkus log level configuration not applied on tests

### ISSUE THREAD

**Issue by oscarfh** [quarkusio/quarkus#8574]
**Describe the bug**
Setting the configuration key to adjust log level is not applied on tests

**Expected behavior**
Log level defined is applied both on running code and on tests

**Actual behavior**
Log level only applied to running code

**To Reproduce**
1.  Download an application from the application generator
2. Add the following lines to your application.properties
```
quarkus.log.level=DEBUG
%test.quarkus.log.level=DEBUG
```
3. Add the following lines to your `ExampleResource` or `ExampleResourceTest`
```
LoggerFactory.getLogger(ExampleResource.class).debug("debug");
LoggerFactory.getLogger(ExampleResource.class).info("info");
```
4. Execute the test
5. Notice how only the info message is written
6. If you added the logging to the ExampleResource class, execute `./gradlew qDev` and notice how both messages appear (debug and info)

**Configuration**
```properties
quarkus.log.level=DEBUG
%test.quarkus.log.level=DEBUG
```

**Screenshots**
(If applicable, add screenshots to help explain your problem.)

**Environment (please complete the following information):**
 - Build tool: gradle
 - IDE: Intellij

**Additional context**
(Add any other context about the problem here.)

**Comment by cemnura** [comment]
Hello @oscarfh,

I have checked and it seems to be working fine.

2020-04-15 11:13:30,927 DEBUG [org.jbo.res.res.i18n] (executor-thread-1) RESTEASY002315: PathInfo: /hello
2020-04-15 11:13:30,928 DEBUG [org.my.gro.MyResource] (executor-thread-1) debug
2020-04-15 11:13:30,928 INFO  [org.my.gro.MyResource] (executor-thread-1) info

**Comment by oscarfh** [comment]
Hi @cemnura ,
Thanks for your reply. This is very strange.
My intellij builds and run tests using Gradle (not Intellij). (On settings > Build, Execution, Deployment > build tools > Gradle). Is yours using Intellij? I could not make this option work.

I found this issue: https://github.com/quarkusio/quarkus/issues/2491 and I made it work. I will make a PR in the documentation with the vertx gradle of https://github.com/quarkusio/quarkus/blob/544773f4c97cbebd0e055d069970fc3e519a9a4a/docs/src/main/asciidoc/logging.adoc#how-to-configure-logging-for-quarkustest

Nevertheless, it would be cool if you could tell me if you are using gradle or intellij to run your tests inside the IDE.  If I try, I get
```
java.lang.RuntimeException: java.lang.RuntimeException: The test class class org.acme.ExampleResourceTest is not located in any of the directories [classes/scala/native-test, classes/java/test, classes/scala/test, classes/java/native-test, /test-classes, bin/test, classes/kotlin/test, classes/kotlin/native-test]
```

**Comment by cemnura** [comment]
Hello @oscarfh,

I ran the test directly using gradlew from the terminal in intellij

**Comment by oscarfh** [comment]
Hello @cemnura 
Ok, thanks a lot for your feedback!

**Comment by oscarfh** [comment]
Hello @cemnura 
Ok, thanks a lot for your feedback, did you use the `-Djava.util.logging.manager=org.jboss.logmanager.LogManager` parameter?

**Comment by cemnura** [comment]
@oscarfh  No I did not use any such parameter. I used  ``./gradlew qDev`` and made a http GET to from the browser.

**Comment by cemnura** [comment]
@oscarfh what ``LoggerFactory`` did you import?

**Comment by oscarfh** [comment]
@cemnura 
Sorry, I was in a meeting.
Yes, with qDev it works (step to reproduce number 6).
Please run the test located in ExampleResourceTest, can you see the debug message?

I added the same that is generated on the pom in the maven version: `org.jboss.logmanager.LogManager`.

If you confirm that you do not see the message when debugging, I will create a PR to code.quarkus code in order to add the correct gradle directives to make it work.

**Comment by cemnura** [comment]
@oscarfh Yes I had the same problem. I was not able to print the debug message during ``./gradlew test -debug``. This appears to be a gradle configuration problem rather then a quarkus or intellij problem. Would you agree?

**Comment by oscarfh** [comment]
I kind of agree. It can be fixed by setting the systemProperty for tests on the build.gradle.
My PR should fix it, I am working now on getting it to build.

**Comment by aloubyansky** [comment]
Fixed by #8594
